test/index.js

```diff
@@ -0,0 +1,244 @@
+const assert = require('assert');
+const helpers = require('../helpers');
+
+describe('Apostrophe-a3-schema-exporter', function() {
+  it('should handle arrays', () => {
+    const field = {
+      name: 'content-links',
+      type: 'array',
+      schema: [
+        {
+          name: 'url',
+          label: 'Url',
+          type: 'url'
+        },
+        {
+          name: 'description',
+          label: 'Description',
+          type: 'string'
+        }
+      ],
+      label: 'Content'
+    };
+    const generatedArray = helpers.handleArray(field);
+
+    assert.deepEqual(generatedArray, {
+      type: 'array',
+      label: 'Content',
+      fields: {
+        add: {
+          url: {
+            label: 'Url',
+            type: 'url'
+          },
+          description: {
+            label: 'Description',
+            type: 'string'
+          }
+        }
+      }
+    });
+  });
+
+  it('should handle tags as an array', () => {
+    const field = {
+      type: 'tags',
+      name: 'tags',
+      label: 'Tags'
+    };
+    const generatedArray = helpers.handleArray(field);
+
+    assert.deepEqual(generatedArray, {
+      type: 'array',
+      label: 'Tags',
+      fields: {
+        add: {
+          tag: {
+            type: 'string',
+            label: 'Tag'
+          }
+        }
+      }
+    });
+  });
+
+  it('should handle areas', async () => {
```

Comment:
```suggestion
  it('should handle areas', function() {
```

apparently we have issues otherwise.

cf. https://mochajs.org/#arrow-functions
cf. https://apostrophe-inc.slack.com/archives/C02KB5BUMU4/p1650477827923089

Reply:
Fixed